### PR TITLE
feat: disabled role reaction

### PR DESCRIPTION
**Pull request by simonhyll**
Disables the role reaction as per #126

### CODE REVIEW

File: src/utils/reactionHandler.ts [hunk 1]
```diff
@@ -8,9 +8,8 @@ import {
     User,
 } from 'discord.js';
 import { REACTION_ROLE, REACTION_ROLE_CHANNEL } from '../config';
-import { wrap_in_embed } from './embed_helpers';
 
-export async function hasPermission(
+export async function hasPermission (
```
Comment (lorenzolewis) [inline src/utils/reactionHandler.ts:12]: Same comment about whitespace

**Review by lorenzolewis — CHANGES_REQUESTED** [review]
Are the updates in `messageCreate` and `messageReactionAdd` meant to be in this PR? The extra whitespace after the function name is throwing me off 😅

File: src/utils/reactionHandler.ts [hunk 2]
```diff
@@ -82,8 +81,6 @@ export async function sendReactionRoleMessage(client: Client) {
         messageArray.push(
```
[the remark this answers is not in the document]
Comment (simonhyll) [reply src/utils/reactionHandler.ts:81]: Sounds good to me :)

File: src/events/messageCreate.ts [hunk 3]
```diff
@@ -97,7 +96,7 @@ export default event({
     },
 });
 
-async function send_instruction_message(thread: ThreadChannel) {
+async function send_instruction_message (thread: ThreadChannel) {
```
[the remark this answers is not in the document]
Comment (simonhyll) [reply src/events/messageCreate.ts:99]: VSCode decided to use the wrong formatter, for reasons >.<

File: src/utils/reactionHandler.ts [hunk 4]
```diff
@@ -106,58 +103,6 @@ export async function sendReactionRoleMessage(client: Client) {
             message = await channel.send(messageBody);
             console.debug('Message sent');
         }
```
[the remark this answers is not in the document]
Comment (simonhyll) [reply src/utils/reactionHandler.ts:105]: I mean they kinda get that sort of info from the new start page thingie, so not sure how impactful that'd be, but I can certainly make it say whatever you want it to say :)

[the remark this answers is not in the document]
Comment (lorenzolewis) [reply src/utils/reactionHandler.ts:110]: That makes sense, let's keep it as this then :)

**Review by lorenzolewis — APPROVED** [review]

**Pull request merged**